### PR TITLE
feat: add legal notice and training

### PR DESCRIPTION
**Pull request by ArchiFleKs**

### CODE REVIEW

File: data/en/formations.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - item: Deployment tools
+      - item: Cluster operation and hardening
+      - item: Main admin API objects
+      - item: Prepare [CKA exam](https://github.com/cncf/curriculum/blob/master/CKA_Curriculum_V1.17.pdf)
+    btnText     : Contact Us
+    btnURL      : "https://particule.io/en#contact"
+    color       : kubernetes
+
+  - name        : Kubernetes - AIO
+    price       : 5 jours
```
Comment (rguichard) [inline data/en/formations.yml]:
```suggestion
    price       : 5 days
```

File: data/en/formations.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      - item: Orchestrate resources on an OpenStack Cloud
+    btnText     : Contact Us
+    btnURL      : "https://particule.io/en#contact"
+    color       : openstack
+
+  - name        : OpenStack Operator
+    price       : 3 days
+    description : Deploy OpenStack with OpenStack-Ansible
+    featurelist :
+      - item: Reference architecture
```
Comment (rguichard) [inline data/en/formations.yml:60]:
```suggestion
      - item: Architecture reference 
 ```

File: data/en/legal.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+      75008 Paris
+
+      SAS au capital social: 1000,00€
+
+      RCS Paris : 879 608 834
+
+      TVA FR: FR 05 879 608 834
+
+      Président: Romain Guichard
```
Comment (rguichard) [inline data/en/legal.yml]:
```suggestion
      Chairman: Romain Guichard
```

File: data/en/legal.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+      Directeur Général: Kevin Lefevre
```
Comment (rguichard) [inline data/en/legal.yml]:
```suggestion
      Executive Director: Kevin Lefevre
```

File: data/fr/formations.yml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      - item: Bonnes pratiques
+      - item: Preparation [CKAD](https://github.com/cncf/curriculum/blob/master/CKAD_Curriculum_V1.17.pdf)
+      - item: Support de cours [HTML](https://particule.io/formations/kubernetes-user.fr.html)/[PDF](https://particule.io/formations/kubernetes-user.fr.html?print-pdf)
+    btnText     : Nous contacter
+    btnURL      : "https://particule.io#contact"
+    color       : kubernetes
+
+  - name        : Kubernetes Opérateur
+    price       : 3 jours
+    description : Deployer et administrer un cluster Kubernetes
```
Comment (rguichard) [inline data/fr/formations.yml]:
```suggestion
    description : Déployer et administrer un cluster Kubernetes
```

File: data/fr/formations.yml [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+      - item: Combine la formation utilisateur et opérateur
+      - item: Prepare au [CKA](https://github.com/cncf/curriculum/blob/master/CKA_Curriculum_V1.17.pdf) et [CKAD](https://github.com/cncf/curriculum/blob/master/CKAD_Curriculum_V1.17.pdf)
+      - item: Support de cours [HTML](https://particule.io/formations/kubernetes-full.fr.html)/[PDF](https://particule.io/formations/kubernetes-full.fr.html?print-pdf)
+    btnText     : Nous contacter
+    btnURL      : "https://particule.io#contact"
+    color       : kubernetes
+
+  - name        : OpenStack Utilisateur
+    price       : 3 jours
+    description : Se familiariser avec les API et le développent d'application Cloud Ready
```
Comment (rguichard) [inline data/fr/formations.yml]:
```suggestion
    description : Se familiariser avec les API et le développement d'applications Cloud Ready
```

File: data/fr/formations.yml [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    featurelist :
+      - item: Découvrir OpenStack et manipuler les différents services
+      - item: Connaître le fonctionnement du projet OpenStack et ses possibilités
+      - item: Manipuler l'API (Application Programming Interface), la CLI (Command Line Interface) et le Dashboard
+      - item: Être capable de définir, déployer et maintenir une infrastructure sur le cloud
```
Comment (rguichard) [inline data/fr/formations.yml]:
```suggestion
      - item: Être capable de définir, déployer et maintenir une infrastructure OpenStack
```

File: data/fr/formations.yml [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+item  :
+  - name        : Kubernetes Utilisateur
+    price       : 3 jours
+    description : Se familiariser avec les API et préparer le CKAD
+    featurelist :
+      - item: Découverte des concepts
+      - item: Les principaux objets de l'API
+      - item: Concevoir et déployer des applications
+      - item: Bonnes pratiques
+      - item: Preparation [CKAD](https://github.com/cncf/curriculum/blob/master/CKAD_Curriculum_V1.17.pdf)
```
Comment (rguichard) [inline data/fr/formations.yml]:
```suggestion
      - item: Préparation [CKAD](https://github.com/cncf/curriculum/blob/master/CKAD_Curriculum_V1.17.pdf)
```

File: data/fr/formations.yml [hunk 9]
```diff
@@ -0,0 +1,72 @@
+enable  : true
+topTitle: 
+title   : formations.
+subtitle : Nous vous aidons, au travers de nos formations, à appréhender les technologies Cloud Native. Nos supports de formations sont entièrement [open-sourcés](https://github.com/particuleio/formations).
```
Comment (rguichard) [inline data/fr/formations.yml]:
```suggestion
subtitle : Nous vous aidons, au travers de nos formations, à appréhender les technologies Cloud Native. Nos supports de formations sont entièrement [Open Source](https://github.com/particuleio/formations).
```

File: data/en/formations.yml (hunk 2 above)
Comment (ArchiFleKs) [reply data/en/formations.yml:60]: Actually this is correct

**Pull request merged**